assets/camel.yaml

```diff
@@ -0,0 +1,27 @@
+---
+
+- type: model
+  name: Camel-5B
+  organization: Writer
+  description: Camel-5B is an instruction-following large language model tailored
+    for advanced NLP and comprehension capabilities.
+  created_date:
+    value: 2023-04-01
+    explanation: The model was stated to be published in April, but which day is
+      not specified on the website.
+  url: https://chatcamel.vercel.app/
+  model_card: https://huggingface.co/Writer/camel-5b-hf
+  modality: text
+  analysis: ''
+  size: 5B parameters
+  dependencies: [Palmyra-Base, Camel dataset]
```

Comment:
Do we have palmyra already? 

Also, we should just have a single asset for palymra, and in that asset we can include the sizes (i.e. the base, large, etc.)

Reply:
I don't think Palmyra is in the ecosystem graph. I'll change the name of the dependency here and add in the asset.